test/unit/helper-specs.js

```diff
@@ -89,4 +89,15 @@ describe("Helpers", function () {
       logger.warn.called.should.equal(false);
     });
   });
-});
+  describe('xcode version', function () {
+    it('parses xcode version with space', function () {
+      var expected = "/Applications/ Xcode 6.1.1.app/Contents/Developer";
+      var actual = "/Applications/\\ Xcode\\ 6.1.1.app/Contents/Developer"
+      if(helpers.escapeSpace(expected).should.equal(actual)) {
```

Comment:
just looking at this test. I think there might be a small misunderstanding about how this works. when you do this:

```
helpers.escapeSpace(expected).should.equal(actual)
```

it will throw if they are not equal. So you don't need to have this conditional, and you don't need to throw anything yourself.

Also, you'll want to switch around your usage of 'actual' and 'expected'. You are escaping the 'actual' value, so that it matches the 'expected' value (i.e., the 'expected' value is the final value we are testing against, the one that is the result of the operation).